packages/tizen_bundle/README.md

```diff
@@ -0,0 +1,133 @@
+# tizen_bundle
+
+Tizen bundle APIs.
+
+## Usage
+
+To use this package, add `tizen_bundle` as a dependency in your `pubspec.yaml` file.
+
+```yaml
+depenedencies:
+  tizen_bundle:
+
+```
+
+## Adding Content to a Bundle
```

Comment:
Please use `###` (h3) if the sections are parts of the usage and also do not capitalize the first letters of the title.

Reply:
Done